sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -64,16 +65,33 @@ export class JsonRpcProvider extends Provider {
   }
 
   // Transactions
+  async getTransactionWithEffects(
```

Comment:
@Jibz1 Could you replace the usage of `getTransaction` with this new function `getTransactionWithEffects`? This will give you the `gas used`. We should also display the `created objects` and `mutated objects` section using the effects content. Please see mocks/data/transaction.json to see some examples

Reply:
@666lcz Ok, let me know when the restart is done